packages/firebase_core/firebase_core/android/global-config.gradle

```diff
@@ -0,0 +1,6 @@
+ext {
+    compileSdk=35
```

Comment:
this shouldn't be in any plugin, it is should be outside like maybe in the script/ directory